capabilities/communication/send-message/profile.supr

```diff
@@ -1,28 +1,54 @@
+"""
+Send Message
+Send message via IM such as Messanger, Slack or MS Teams.
+"""
+
 name = "communication/send-message"
-version = "1.0.1"
+version = "1.1.0"
 
 """
 Send Message
-  Sends message via IM such as Messanger, Slack or MS Teams...
+Sends message to one destination
 """
 usecase SendMessage {
   input {
-    "Destination
-      Representation of a messaging destination. Can be ID or channel name, depending on the provider capability."
+    """
+    Destination
+    Representation of a messaging destination. Can be ID or channel name, depending on the provider capability.
+    """
     destination! string!
 
-    "Text
-      Text of the message"
+    """
+    Text
+    Text of the message
+    """
     text! string!
   }
 
   result {
-    "Destination ID
-      Identifier of the destination where the message was sent."
-    destinationId
+    """"
+    Destination
+    Identifier of the destination where the message was sent.
+    """
+    destination
```

Comment:
I would consider this breaking change and bump major version of the profile.

Reply:
Yeah it should be.